__tests__/app.test.js

```diff
@@ -30,20 +30,37 @@ describe('/api/topics', () => {
 
 describe('/api/articles/:article_id', () => {
     test('GET request responds with an array of topic objects, each of which has slug and description properties', () => {
```

Comment:
I missed this last time - looks like you've fallen victim to copy paste! Be vigilant when doing this!
Update to be more reflective of what this test covers